Formula/eless.rb

```diff
@@ -0,0 +1,23 @@
+class Eless < Formula
+  desc "Better 'less' using Emacs view-mode and Bash"
+  homepage "https://eless.scripter.co"
+  url "https://github.com/kaushalmodi/eless/archive/v0.5.tar.gz"
+  sha256 "b4da2c7c223996681bb951d50e0d542d8df04baf765c16412fa0848cdb2b3a3d"
+
+  head do
```

Comment:
Kill the block and just use `head "https://github.com/kaushalmodi/eless.git"` under `sha256` above.